latest/ug/troubleshooting/troubleshooting.adoc

```diff
@@ -314,64 +314,57 @@ If your node enters a `NotReady` status, this likely indicates that the node is
 
 For Amazon EKS optimized Windows AMIs, there's no reservation for compute resources specified by default in the `kubelet` configuration. To help prevent resource issues, you can reserve compute resources for system processes by providing the `kubelet` with configuration values for https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#kube-reserved[kube-reserved] and/or https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#system-reserved[system-reserved]. You do this using the `-KubeletExtraArgs` command-line parameter in the bootstrap script. For more information, see https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/[Reserve Compute Resources for System Daemons] in the Kubernetes documentation and <<bootstrap-script-configuration-parameters>> in this user guide.
 
-[#troubleshoot-cni]
-== CNI log collection tool
+[#log-collector]
+== EKS Log Collector
 
-The Amazon VPC CNI plugin for Kubernetes has its own troubleshooting script that is available on nodes at `/opt/cni/bin/aws-cni-support.sh`. You can use the script to collect diagnostic logs for support cases and general troubleshooting.
+To troubleshoot issue with Amazon EKS nodes, there is a pre-built script available on nodes located at `/etc/eks/log-collector-script/eks-log-collector.sh`. You can use the script to collect diagnostic logs for support cases and general troubleshooting.
 
 Use the following command to run the script on your node:
 
 [source,bash,subs="verbatim,attributes"]
 ----
-sudo bash /opt/cni/bin/aws-cni-support.sh
+sudo bash /etc/eks/log-collector-script/eks-log-collector.sh
 ----
 
 [NOTE]
 ====
 
-If the script is not present at that location, then the CNI container failed to run. You can manually download and run the script with the following command:
+If the script is not present at that location. You can manually download and run the script with the following command:
 
 [source,bash,subs="verbatim,attributes"]
 ----
-curl -O https://raw.githubusercontent.com/awslabs/amazon-eks-ami/master/log-collector-script/linux/eks-log-collector.sh
```

Comment:
Cross ref: https://github.com/awslabs/amazon-eks-ami/pull/2121

Reply:
fyi, the script `aws-cni-support.sh` is actually the same as `eks-log-collector.sh` from `awslabs/amazon-eks-ami`

https://github.com/aws/amazon-vpc-cni-k8s/blob/master/Makefile#L304-L315